troposphere/static/js/components/modals/image_version/ImageVersionEditModal.react.js

```diff
@@ -159,9 +167,23 @@ export default React.createClass({
     // Render
     // ------
     //
+    validateName: function(image, name) {
+        let orig_version = this.props.version,
+        versions = image.get('versions');
 
+        name = $.trim(name).toLowerCase();
+        versions = versions.filter(function(version) {
+            return (version.id !== orig_version.id && version.name.toLowerCase() === name);
+        });
```

Comment:
I saw some errors related to `toLowerCase()` generated by your development server:
```
TypeError: Cannot read property 'toLowerCase' of undefined
```

I don't know if that was because `validateName` has `name` argument passed in as `undefined` (or not passed) or `version` in Line 175-177 was `undefined`. Maybe it was just generated during development. But it makes me wonder if we might want to make the more cautious: 

```javascript
         versions = versions.filter(function(version) {
              return (version.id !== orig_version.id && 
                 (version.name && version.name.toLowerCase() === name));
          });
```
I am not sure if I'm being overly paranoid here. Let's discuss.